libs/validate_results.py

```diff
@@ -0,0 +1,112 @@
+import csv 
+import sys
+
+from libs.us_state_abbrev import us_state_abbrev
+
+"""
+A set of functions to validate the datasets we prepare
```

Comment:
Thoughts on renaming this file `libs/validate_dod_results.py` for now?  Better to just be more explicit about that for the time being, since we don't quite validate all of the datasets just yet here.

Also, we have some of this logic in `libs/datasets/dataset_utils.py` (definitely not all of it!).  do you think we could combine the validation logic functions from `libs/datasets/dataset_utils.py` with these? maybe `libs/datasets/validation.py`?